test/cmdlineTests/asm_json_import_all_valid_flags/output

```diff
@@ -0,0 +1,91 @@
+{
+    "contracts":
+    {
+        "<stdin>":
+        {
+            "asm":
+            {
+                ".code":
+                [
+                    {
+                        "begin": 0,
+                        "end": 0,
+                        "name": "PUSH",
+                        "source": -1,
+                        "value": "0"
+                    }
+                ],
+                ".data":
+                {
+                    "0":
+                    {
+                        ".code":
+                        [
+                            {
+                                "begin": 0,
+                                "end": 0,
+                                "name": "PUSH",
+                                "source": -1,
+                                "value": "1"
+                            }
+                        ]
+                    }
+                },
+                "sourceList":
+                [
+                    "contract.sol",
+                    "#utility.yul"
+                ]
+            },
+            "bin": "5ffe",
+            "bin-runtime": "6001",
+            "opcodes": "PUSH0 INVALID ",
```

Comment:
Isn't this supposed to be `PUSH0 PUSH1`?

Reply:
Good question. I'm not sure. Summoning @ekpyron here ;)

Reply:
What would ``PUSH0 PUSH1`` even mean (without the push data for the push1?) :-)? I guess the ``opcodes`` field here is the disassembled version of ``bin``, which has ``PUSH0 INVALID`` in it...